examples/resources/edgecast_waf_bot_manager/resource.tf

```diff
@@ -0,0 +1,60 @@
XJ0eT0ib2c6dGl0bGUiLz4KICAgIDxtZXRhIGNvbnRlbnQ9IndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xIiBuYW1lPSJ2aWV3cG9ydCIvPgogICAgPHN0eWxlPgogICAgICBib2R5IHsKICAgICAgICBmb250LWZhbWlseTogc2Fucy1zZXJpZjsKICAgICAgICBsaW5lLWhlaWdodDogMS4yOwogICAgICAgIGZvbnQtc2l6ZTogMThweDsKICAgICAgfQogICAgICBzZWN0aW9uIHsKICAgICAgICBtYXJnaW46IDAgYXV0bzsKICAgICAgICBtYXJnaW4tdG9wOiAxMzBweDsKICAgICAgICB3aWR0aDogNzUlOwogICAgICB9CiAgICAgIGgxIHsKICAgICAgICBmb250LXNpemU6IDUwcHg7CiAgICAgICAgbGluZS1oZWlnaHQ6IDQ1cHg7CiAgICAgICAgZm9udC13ZWlnaHQ6IDcwMDsKICAgICAgICBtYXJnaW4tYm90dG9tOiA3NXB4OwogICAgICAgIHdoaXRlLXNwYWNlOiBub3dyYXA7CiAgICAgIH0KICAgICAgcCB7CiAgICAgICAgbWFyZ2luLWJvdHRvbTogMTBweDsKICAgICAgfQogICAgICBzbWFsbCB7CiAgICAgICAgZm9udC1zaXplOiA4MCU7CiAgICAgICAgY29sb3I6ICMzMzM7CiAgICAgIH0KICAgICAgZm9vdGVyIHsKICAgICAgICBwb3NpdGlvbjogZml4ZWQ7CiAgICAgICAgYm90dG9tOiAwOwogICAgICAgIGxlZnQ6IDA7CiAgICAgICAgcGFkZGluZzogLjdyZW0gMCAuN3JlbSA0cmVtOwogICAgICAgIHdpZHRoOiAxMDAlOwogICAgICAgIGJhY2tncm91bmQ6ICBJbmRpZ287CiAgICAgIH0KICAgICAgZm9vdGVyIGEgewogICAgICAgIGNvbG9yOiB3aGl0ZTsKICAgICAgICBtYXJnaW4tbGVmdDogNDBweDsKICAgICAgICB0ZXh0LWRlY29yYXRpb246IG5vbmU7CiAgICAgIH0KICAgICAgLmQtbm9uZSB7CiAgICAgICAgZGlzcGxheTogbm9uZSAhaW1wb3J0YW50OwogICAgICB9CiAgICAgIC5zZWN0aW9uLWVycm9yIHsKICAgICAgICBjb2xvcjogI2JkMjQyNjsKICAgICAgfQogICAgICAubG9hZGluZyB7CiAgICAgICAgYW5pbWF0aW9uOiAzcyBpbmZpbml0ZSBzbGlkZWluOwogICAgICB9CiAgICAgIEBrZXlmcmFtZXMgc2xpZGVpbiB7CiAgICAgICAgZnJvbSB7CiAgICAgICAgICBtYXJnaW4tbGVmdDogMTAlOwogICAgICAgICAgY29sb3I6IHJnYigxMzQsIDUxLCAyNTUpOwogICAgICAgIH0KCiAgICAgICAgMzAlIHsKICAgICAgICAgIGNvbG9yOiByZ2IoMTM0LCA1MSwgMjU1KTsKICAgICAgICB9CgogICAgICAgIHRvIHsKICAgICAgICAgIG1hcmdpbi1sZWZ0OiAwJTsKICAgICAgICAgIGNvbG9yOiBibGFjazsKICAgICAgICB9CiAgICAgIH0KICAgIDwvc3R5bGU+CiAgPC9oZWFkPgoKICA8Ym9keSBvbmxvYWQ9Im9ubG9hZENvb2tpZUNoZWNrKCkiPgogICAge3tCT1RfSlN9fQogICAgPHNlY3Rpb24+CiAgICAgIDxoMT4KICAgICAgICBWYWxpZGF0aW5nIHlvdXIgYnJvd3NlciAtIGN1c3RvbSBjaGFsbGVuZ2UgcGFnZQogICAgICAgIDxzcGFuIGNsYXNzPSJsb2FkaW5nIj4uPC9zcGFuPjxzcGFuIGNsYXNzPSJsb2FkaW5nIj4uPC9zcGFuPjxzcGFuIGNsYXNzPSJsb2FkaW5nIj4uPC9zcGFuPgogICAgICA8L2gxPgoKICAgICAgPG5vc2NyaXB0PgogICAgICAgIDxoNCBjbGFzcz0ic2VjdGlvbi1lcnJvciI+UGxlYXNlIHR1cm4gSmF2YVNjcmlwdCBvbiBhbmQgcmVsb2FkIHRoZSBwYWdlLjwvaDQ+CiAgICAgIDwvbm9zY3JpcHQ+CgogICAgICA8ZGl2IGlkPSJjb29raWUtZXJyb3IiIGNsYXNzPSJkLW5vbmUiPgogICAgICAgIDxoNCBjbGFzcz0ic2VjdGlvbi1lcnJvciI+UGxlYXNlIGVuYWJsZSBjb29raWVzIGFuZCByZWxvYWQgdGhlIHBhZ2UuPC9oND4KICAgICAgPC9kaXY+CgogICAgICA8cD5UaGlzIG1heSB0YWtlIHVwIHRvIDUgc2Vjb25kczwvcD4KCiAgICAgIDxzbWFsbD5FdmVudCBJRDoge3tFVkVOVF9JRH19PC9zbWFsbD4KICAgIDwvc2VjdGlvbj4KCiAgICA8Zm9vdGVyPgogICAgICA8cD4KICAgICAgICA8YSBocmVmPSJodHRwczovL3d3dy5lZGdlY2FzdC5jb20vc2VjdXJpdHkvIj5Qb3dlcmVkIGJ5IEVkZ2lvPC9hPgogICAgICA8L3A+CiAgICA8L2Zvb3Rlcj4KICA8L2JvZHk+CiAgPHNjcmlwdD4KICAgIGZ1bmN0aW9uIG9ubG9hZENvb2tpZUNoZWNrKCkgewogICAgICBpZiAoIW5hdmlnYXRvci5jb29raWVFbmFibGVkKSB7CiAgICAgICAgZG9jdW1lbnQuZ2V0RWxlbWVudEJ5SWQoJ2Nvb2tpZS1lcnJvcicpLmNsYXNzTGlzdC5yZW1vdmUoJ2Qtbm9uZScpOwogICAgICB9CiAgICB9CiAgPC9zY3JpcHQ+CjwvaHRtbD4="
```

Comment:
I think it would be best to store the custom html as a file (like we do with RE policies) and read it in from the file. It has to be base64 encoded so we can use this tf function: https://developer.hashicorp.com/terraform/language/functions/base64encode